testutil/helpers.go

```diff
@@ -9,10 +9,22 @@ import (
 	"testing"
 )
 
-var rootEnabled bool
+var (
+	rootEnabled bool
+	runParallel bool
+)
 
 func init() {
 	flag.BoolVar(&rootEnabled, "test.root", false, "enable tests that require root")
+	flag.BoolVar(&runParallel, "test.concurrent", false, "run elligible tests concurrently")
```

Comment:
What is wrong with the existing flag? This makes things a little confusing to add flags on top of flags.